mailbox/store/src/test/java/org/apache/james/mailbox/store/search/AbstractMessageSearchIndexTest.java

```diff
@@ -309,20 +309,46 @@ public void searchingMessageInMultipleMailboxShouldNotReturnLessMessageThanLimit
         List<MessageId> result = messageSearchIndex.search(session,
             ImmutableList.of(mailbox2.getMailboxId(), mailbox.getMailboxId()),
             searchQuery,
-            limit);
+            limit).getMessageIds();
 
         assertThat(result)
             .hasSize(limit);
     }
 
+
+    @Test
+    public void searchingMessageInMultipleMailboxShouldReturnTotalNumberOfMessage() throws Exception {
```

Comment:
Can we also have a test thatndo not require message ID ?

Reply:
Do we need it? we searched in multiple mailboxes so it still makes sense with request messageID

Reply:
No you are right. total is used only with multi mailbox search. We can skip it for now.